core/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admi
 WORKDIR /app
 RUN pip install -U pip && \
     pip install --no-cache-dir . &&\
-    python3 -c "import nltk; nltk.download('punkt')" &&\
+    python3 -c "import nltk; nltk.download('punkt');nltk.download('averaged_perceptron_tagger')" &&\
```

Comment:
I got some errors on docker on windows when upload a file. The error tells that missing package. Yeah maybe could be outside the PR but someone need to add it.

Reply:
It is useful to do that download durgin docker build, otherwise it is launched at every `docker-compose up`

(btw that line is already in core, from another PR by @varrialeciro, just ignore it)